apps/leo_manager/src/leo_manager_api.erl

```diff
@@ -2408,6 +2408,19 @@ call_gateway_api(Method, Args) ->
 %% @doc Join a cluster (MDC-Replication)
 -spec(join_cluster([atom()], #?SYSTEM_CONF{}) ->
              {ok, #?SYSTEM_CONF{}} | {error, any()}).
+%% @doc Convert System Conf from ~1.3.2 cluster
+join_cluster(RemoteManagerNodes, #system_conf_2{} = SystemConf) ->
```

Comment:
@windkit I recommend you use [leo_cluster_tbl_conf:transform/1](https://github.com/leo-project/leo_redundant_manager/blob/develop/src/leo_cluster_tbl_conf.erl#L237) to convert to the correct record in case of not current `#?SYSTEM_CONF{}`.